subprojects/core/src/main/java/org/gradle/internal/taskgraph/CalculateTaskGraphBuildOperationType.java

```diff
@@ -30,7 +30,11 @@
 
     @UsedByScanPlugin
     public interface Details {
-
+        /**
+         * The build path the calculated task graph belongs too)
```

Comment:
dangling `)` on the end.

Can you add a blank line between this and `@since` too please.